services/core/shared/core.js

```diff
@@ -329,6 +329,36 @@ const getBlocks = async params => {
 const setReadyStatus = status => { readyStatus = status; };
 const getReadyStatus = () => readyStatus;
 
+const calcExpDecay = (NUM_BLOCKS, DECAY_RATE) => {
+	const expDecay = (1 - Math.pow(1 - DECAY_RATE, 1 / NUM_BLOCKS)).toFixed(5);
+	return expDecay;
+};
```

Comment:
1. `calcExpDecay` method can be defined as an inner function within `EMAcalc`. `calcExpDecay` is a one-liner and not used anywhere else.
2. Let's keep the variable names consistent with the ones defined in the config file for the `calcExpDecay` method. `NUM_BLOCKS` to `emaBatchSize` and `DECAY_RATE` to be `emaDecayRate`.

Reply:
changes has been done as requested